duckduckgo/duckduckgo-autofill
# Prevent in-context signup showing on local or invalid domain pages

**Reviewer:** @shakyShane @GioSensation 
**Asana:** https://app.asana.com/0/0/1204038361216231/f

## Description
Restrict in-context signup from running on local pages with valid TLD. This branch also updates the in-context signup tests to run on a real domain, mocked in puppeteer.

## Steps to test
1. Build update into extension
2. Run local server and go to `email-autofill.html` integration test file
3. Confirm that in-context signup does not display
4. Go to https://duckduckgo.com/newsletter
5. Confirm that in-context signup does display
